lib/encodings.js

```diff
@@ -89,7 +89,9 @@ Encodings.encodingMethodDefaultOptions = {
   gzip: {},
   deflate: {},
   br: {
-    [zlib.constants.BROTLI_PARAM_QUALITY]: 4
+    params: {
```

Comment:
With this change, https://github.com/koajs/compress/blob/fc8fca07f7f3c60345103518ff7f60d8a3c78ce6/lib/index.js#L41-L42 may need to change too.

---

As the PR is now, `options['br']` will shallow merge, so if a consumer provides
```js
koaCompress({
  br: {
    params: {
      [zlib.constants.BROTLI_PARAM_MODE]: zlib.constants.BROTLI_MODE_TEXT
    },
  },
});
```

I think the final object sent to `zlib.createBrotliCompress` ends up being
```js
{
  params: {
    [zlib.constants.BROTLI_PARAM_MODE]: zlib.constants.BROTLI_MODE_TEXT,
    // the koa-compress default of [zlib.constants.BROTLI_PARAM_QUALITY]: 4 is no longer passed in
  },
}
```

It can go either way but users might expect the brotli options to deep merge here

Reply:
> I think the final object sent to zlib.createBrotliCompress ends up being

Correct. 

> users might expect the brotli options to deep merge here

True, I'll happily defer to maintainers here on what the koa community expects.